Gemfile

```diff
@@ -15,6 +15,9 @@ group :development do
   gem 'activesupport', '4.2.9'
   gem 'github_changelog_generator', '~> 1.14'
   gem 'pry-byebug', '~> 3.4'
+  # if you want to test out the pdk console you will need puppet-debugger gem
+  # the unit tests don't actually call the debugger, so this is not needed
+  # gem 'puppet-debugger' # only needed in bundler sandbox
```

Comment:
Won't stop merging.  @rodjek , is the pattern for "optional" gems to guard them behind an environment variable, or prompt users to use the `Gemfile.local` file?

Reply:
If left enabled the license checker fails

Reply:
If left enabled the license checker fails

Reply:
I am just removing this in latest push.